### ISSUE TITLE
Ore Machine Permanently broken

### ISSUE THREAD

**Issue by ghost** [HippieStationArchive/HippieStation13-2#434]
In-Game report from lordquailius:
Revision info: Server revision ebeb3916fa4e7fcb4dd294d9cf0725e8ee805895 compiled on: 2017-03-07

>General description of the issue
Ore Machine just stopped fucking working and absolutely nothing fixed it. Not even admins spawning in a new one fixed it, it just went and fucked itself

>What you expected to happen
The machine to work

>What actually happened

It didn't
>Steps to reproduce if possible

Damage the ore machine

**Comment by GuyonBroadway** [comment]
Did you put the ore to the north of the machine or to the west?

**Comment by Pyko1** [comment]
Yeah most likely dir issue, should make it so that machines retain their dir when upgraded.
